compiler/grainformat/comment_utils.re

```diff
@@ -0,0 +1,519 @@
+open Grain_diagnostics;
+module Doc = Res_doc;
+
+let rec get_comments_on_line = (line: int, comments) =>
+  if (List.length(comments) == 0) {
+    [];
+  } else {
+    let c: Grain_parsing.Parsetree.comment = List.hd(comments);
```

Comment:
Maybe it's just a personal thing, but I prefer to have the type annotations on the arguments to the functions (if I have any).

Reply:
Good point, adding them!